pkg/reconciler/autoscaling/kpa/scaler.go

```diff
@@ -307,6 +308,17 @@ func (ks *scaler) scale(ctx context.Context, pa *pav1alpha1.PodAutoscaler, sks *
 	}
 
 	min, max := pa.ScaleBounds()
+	initialScale := kparesources.GetInitialScale(config.FromContext(ctx).Autoscaler, pa)
+	sw := config.FromContext(ctx).Autoscaler.StableWindow
+	hasBeenActiveFor := pa.Status.ScaleTargetInitializedFor(time.Now())
+	if initialScale > 1 && hasBeenActiveFor < sw {
```

Comment:
If initialScale == 1 which is the default case, then we will end up changing `min` 0 to 1, and the rest of the logic doesn't work properly because we are adjusting the `desiredScale` based on the new `min`.